src/main.c

```diff
@@ -177,7 +177,6 @@ int main(int argc, char* argv[]) {
   jv program_arguments = jv_array();
 
 #ifdef WIN32
-  SetConsoleOutputCP(CP_UTF8);
```

Comment:
Is this something we should generally do if `isatty(STDOUT_FILENO)`, or should we never do this?